scss/_variables.scss

```diff
@@ -110,6 +110,8 @@
     */
 
     $blueGreen  : #0c99bd;
+    $Dark_Gray: hsl(0, 0%, 7%);
+    $GREEN-BUTTON-BASKET: hsl(145, 63%, 49%);
 
```

Comment:
We need to define a variable naming convention as well, all the team should define a new variable in the same structure.